packages/react-components/src/AddressInfo.tsx

```diff
@@ -274,7 +274,7 @@ function renderBalances (props: Props, allAccounts: string[], t: <T = string> (k
                   <div className='faded'>{t('available to be unlocked')}</div>
                   {formatBalance(balancesAll.vestingPerBlock)}
                   <div className='faded'>{t('per block')}</div>
-                  <div className='faded'>{`${t('until block')} ${balancesAll.vestingEndBlock}`}</div>
+                  <div className='faded'>{`${t('until block')} ${balancesAll.vestingEndBlock.toString()}`}</div>
```

Comment:
Was hoping that we could have an end date like the calendar (or at least countdown via `BlockToTime`), however if we do display just the blockNumber, display it via `formatNumber(balancesAll.vestingEndBlock)` to align with the rest of the UI formatting.

Reply:
yes adding. This needed to be merged to show remaining blocks BlockToTime https://github.com/polkadot-js/api/pull/2566/files

Reply:
Then this needs to be added in this line. 

`<div className='faded'>{`${t('until block')} ${formatNumber(balancesAll.vestingEndBlock)} ${t('on')} `} <BlockToTime blocks={balancesAll.vestingRemainingBlocks} /></div>`

Don't want to push this until https://github.com/polkadot-js/api/pull/2566/files is merged so to not break the PR

Reply:
done